content/posts/2024-05-06_trunk_and_tidbits-april_2024/index.md

```diff
@@ -0,0 +1,91 @@
+
+## iOS
+
+- A new release with improved polls, the return of the local timeline, better explore tab, and multiple smaller changes and fixes ([release notes](https://github.com/mastodon/mastodon-ios/releases/tag/2024.4))
+- The iOS app is still undergoing a behind-the-scene large refactor to make it easier to work on. We are focused on improving it’s stability, and adding the most-requested missing features.
+
+## Android
+
+- A new version adding support for the new “filtered notifications” feature that will be released in 4.3 and some small changes and improvements ([release notes](https://github.com/mastodon/mastodon-android/releases/tag/v2.5.0))
+- Working on an experiment on mastodon.social to show a **non-intrusive, easily dismissible** banner to donate to the project, similar to what Signal is doing. This is the first step of a bigger project to try to have the Mastodon ecosystem financed by its users and, if it proves successful, will be slowly expanded with more features (specifically, the ability for donations to go to server admins)
+
+# What’s coming
+
+These items are actively being worked on…
+
+<div class="features-list">
+
+- {{< feature-label name="new feature" >}} Grouped notifications API: with this feature, notifications like boosts, follows or likes on the same post will be grouped if they occur a few hours appart, and will show “X, Y, Z and 12 other accounts liked your post”. The UI is not complete, and we want to experiment with it a bit to see if it works. This required significant effort to implement without causing performance issues, but we think this should make notifications much better! {{< github-pr id=29889 authors="ClearlyClaire" >}}
+- {{< feature-label name="api" >}} A new batch fetch API to get multiple accounts or statuses at once {{< github-pr id=27871 authors="noellabo,ClearlyClaire" >}}
+- {{< feature-label name="performance" >}} Adding support for using `libvips` as our image processing library, which should make image operations more secure, much faster and require less memory {{< github-pr id=30090 authors="Gargron" >}}
+- {{< feature-label name="performance" >}} We are trying to ship OpenTelemetry support in 4.3, to provide server admins with very useful operation data, and to drive our performance work. {{< github-pr id=30130 authors="robbkidd,julianocosta89,renchap" >}}
+- {{< feature-label name="developer" >}} Work is ongoing to move our frontend build system from the deprecated Webpacker Gem to Vite, a more modern, faster and simpler bundler. It will not block the 4.3 release if not ready in time, but is near completion. {{< github-pr id=24981 authors="renchap" >}}
+
+</div>
+
+Recruitment for a new [full-time core developer position](https://jobs.ashbyhq.com/mastodon/de1e4d8a-5639-4b6a-86d7-97bd9c768b02) is still in progress and we have been meeting with amazing people,
```

Comment:
Looks like this is half a sentence? I don't remember if there was anything after the comma